src/shogun/regression/GaussianProcessRegression.cpp

```diff
@@ -76,7 +76,6 @@ bool GaussianProcessRegression::train_machine(std::shared_ptr<Features> data)
 	// check whether given combination of inference method and likelihood
 	// function supports regression
 	require(m_method, "Inference method should not be NULL");
```

Comment:
If you stumble across messages like this, would you mind changing them to "Inference method not set" ? Thx